taipy/core/taipy.py

```diff
@@ -516,22 +521,37 @@ def get_scenarios(cycle: Optional[Cycle] = None, tag: Optional[str] = None) -> L
     Parameters:
          cycle (Optional[Cycle^]): The optional `Cycle^` to filter scenarios by.
          tag (Optional[str]): The optional tag to filter scenarios by.
+         sorted (Optional[bool]): The option to sort scenarios. The default sorting key is name.
+         sort_key (Optional[Literal["name", "id", "creation_date", "tags"]]): The optiononal sort_key to decide upon
+         what key scenarios are sorted.
 
     Returns:
-        The list of scenarios filtered by cycle or tag. If no filtering criteria
-            are provided, this method returns all existing scenarios.
+        The list of scenarios filtered by cycle or tag and optionally sorted by name, id, creation_date or tags.
+        The sorting is in increasing order for dates, in alphabetical order for name and id
+        and in lexographical order for tags.
+        If no filtering criteria are provided, this method returns all existing scenarios.
+        If sorted is set to true, but an incorrect or no sort_key is provided, then the scenarios are sorted by name.
```

Comment:
I would not explain the sorting order here. If necessary, I think we can explain it right at the `sort_key` param description.

Reply:
I was not aware that it is more of a standard practice to include it directly in the parameters. Your changes look good!